src/app/pcasts/controllers/search_episode_controller.py

```diff
@@ -0,0 +1,35 @@
+from . import *
+
+class SearchEpisodeController(AppDevController):
+
+  def get_path(self):
+    return '/search/episodes/<query>'
```

Comment:
I think we are trying to have all endpoints end with a `/` just for convention, there may be some inconsistencies still in the app but I try to fix them when i see them